basic/src/test/java/org/jboss/hal/testsuite/test/runtime/server/configurationchanges/ConfigurationChangesTestCase.java

```diff
@@ -52,15 +52,14 @@
     @Page
     private ConfigurationChangesPage page;
 
-    private static OnlineManagementClient client;
-    private static Operations operations;
+    private static final OnlineManagementClient client = ManagementClientProvider.createOnlineManagementClient();
+    private static final Operations operations = new Operations(client);
+    private static final Administration administration = new Administration(client);
     private static Address serviceAddress;
     private static ConfigurationChangesResource changesResource;
 
     @BeforeClass
-    public static void beforeClass() {
-        client = ManagementClientProvider.createOnlineManagementClient();
-        operations = new Operations(client);
+    public static void beforeClass() throws InterruptedException, TimeoutException, IOException {
```

Comment:
No need for the throws as those exceptions are not thrown from the used methods.